movie_search/models.py

```diff
@@ -14,38 +13,51 @@ class Meta:
         verbose_name_plural = "genres"
 
 class Provider(models.Model):
-    name = models.CharField(max_length=200)
+    name = models.CharField(max_length=200, null=True, blank=True)
     provider_id = models.PositiveSmallIntegerField(default=0)
 
     def __str__(self):
         return f"{self.provider_id}: {self.name}"
 
 
 class Video(models.Model):
-   
-    name = models.CharField(max_length=200)
-    key =  models.CharField(max_length=200)
+    name = models.CharField(max_length=200, null=True, blank=True)
+    key =  models.CharField(max_length=200, null=True, blank=True)
```

Comment:
https://sentry.io/answers/django-difference-between-null-and-blank/#:~:text=Although%20we%20could%20still%20set,or%20an%20empty%20string%20%22%22%20.

<img width="790" alt="image" src="https://github.com/vnoelifant/movie-search-app/assets/387927/4d925ea0-442b-4422-aebe-e4855248bf7c">

+

<img width="721" alt="image" src="https://github.com/vnoelifant/movie-search-app/assets/387927/a0652040-9500-4da5-83a7-21217487417f">